packages/site/src/pages/index.tsx

```diff
@@ -308,7 +328,7 @@ const Index = () => {
     let transactionDetails: Record<string, any> = {
       payload: {
         to: targetAccount,
-        value: transferAmount,
+        value: parseUnits(transferAmount, 'ether'),
         data: '0x'
       },
       account: snapState.accounts[0]?.id || '', // TODO: need to use currently selected snap account
```

Comment:
please update to use the currently selected account in all places (this object is used on ETH transfer)
most of these places where the changes have to made are marked in "TODO:"